docs/04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md

```diff
@@ -80,16 +80,16 @@ kyma deploy --values-file {VALUES_FILE_PATH}
       cpu: 40m
       memory: 200Mi
   ```
+  
+  > **TIP:** You should be fine with increasing the limit to 6Gi. However, if your resources are scarce, try increasing the value gradually in steps of 1Gi.
 
-> **TIP:** You should be fine with increasing the limit to 6Gi. But if your resources are scarce, try increasing the value gradually in steps of 1Gi.
-
-3. If the problem persists, drop labels for the Istio metrics, edit `prometheus-istio server`:
+3. If the problem persists, drop additional labels for the Istio metrics by editing `prometheus-istio server`:
 
   ```bash
   kubectl edit configmap -n kyma-system monitoring-prometheus-istio-server
   ```
 
-4. Edit the values:
+4. Set the following values:
 
   ```yaml
```

Comment:
Please add single spaces at the beginning of lines:
- 94-100
- 104-106
- 110-112
each. I can't add suggestions for these.